src/scss/mixins/_button-base-styles.scss

```diff
@@ -88,4 +58,41 @@
     transform: none !important;
     cursor: not-allowed !important;
   }
+
+  /// INTERACTIVE STATE
+  /// A pseudo-element is used to create the material design
+  /// ripple effect on hover/active
+  &::after {
+    content: '';
+    display: block;
+    inset: -1px; // Covers the border then, too
+    position: absolute;
+    background-position: center;
+    transition: background 0.8s;
+    opacity: 0.2;
+  }
+
+  &:hover::after {
+    background: get-utility-value('bg', 'action-bg-hover')
+      radial-gradient(
+        circle,
+        transparent 1%,
+        get-utility-value('bg', 'action-bg-hover') 1%
+      )
+      center/15000%;
+  }
+
+  &:hover {
+    transform: translateY(-0.15em);
```

Comment:
I was looking at [the material spec](https://material.io/components/buttons) and it doesn't seem like their button does a translate.

Reply:
Ah shoot, my bad. Good catch